routers/name_router.py

```
@@ -0,0 +1,43 @@
+    return NameHandler()
+
+
+@name_router.post("/")
+def create_name(
+    input_name: NameReqDTO, handler: NameHandler = Depends(get_name_handler)
+):
+    return handler.create_name(input_name)
+
+
+@name_router.get("/")
+def get_names(handler: NameHandler = Depends(get_name_handler)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)